src/index.js

```diff
@@ -51,6 +51,10 @@ export default class RestAdapter {
 
         const body = response.body instanceof Object ? JSON.stringify(response.body) : response.body;
 
+        if (res.getHeader('Content-Length') != null) {
```

Comment:
why is it not:
- `res.getHeader('Content-Length') == null` (or `!res.getHeader('Content-Length')`) ?
- `!=` rather than `!==`?

Reply:
`!res.getHeader('Content-Length')` is true for 0
`res.getHeader('Content-Length') == null` is the opposite of what we want
`res.getHeader('Content-Length') !== null` is false for undefined

Reply:
I guess, why do you want to set it only when it's already set? Why is it wrong?

Reply:
I want to correct the header when it's wrong, if consumer don't want to send a content-length header for some reason than is should remain unset.

Reply:
Any idea why it’s wrong?